cmd/cmdutil/defaults.go

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Can we move the `cmdutil` package under `/pkg/cmdutil` ?

Reply:
For sure! f041164